mesonbuild/backend/backends.py

```diff
@@ -69,17 +69,13 @@ def __init__(self, fname, outdir, aliases, strip, install_name_mappings, install
         self.optional = optional
 
 class ExecutableSerialisation:
-    def __init__(self, name, fname, cmd_args, env, is_cross, exe_wrapper,
-                 workdir, extra_paths, capture, needs_exe_wrapper: bool):
-        self.name = name
-        self.fname = fname
+    def __init__(self, cmd_args, env={}, exe_wrapper=None,
```

Comment:
__[Flake8]__ [B006] Do not use mutable data structures for argument defaults. All calls reuse one instance of that data structure, persisting changes between them. ([link](https://sider.review/gh/repos/19784232/pulls/5644#issue-4339556))

<sub>posted by Sider</sub>